docs/source/user/aerodyn/theory_ua.rst

```diff
@@ -465,13 +487,79 @@ Inputs
+Their default values are described in :numref:`UA_AFI_defaults`
 
 See :numref:`ua_notations` for a list of notations and definitions specific to unsteady aerodynamic inputs.
 
 An example of profile data (containing some of the unsteady aerodynamic parameters) is available here
 :download:`(here) <examples/ad_polar_example.dat>`.
 
 
+.. _UA_AFI_defaults:
+
+Calculating Default Airfoil Coefficients
+----------------------------------------
+
+The default value for ``cd0`` is the minimum value of the :math:`C_d` curve between :math:`\pm20` degrees angle of attack. 
+:math:`\alpha_{c_{d0}}` is defined to be the angle of attack where ``cd0`` occurs.
+
+After computing ``cd0``, the :math:`C_n` curve is computed by
+
+.. math::
+       C_{n}(\alpha) = C_l(\alpha) \cos\alpha + \left(C_d(\alpha) - c_{d0}\right) \sin\alpha
+
+The slope of the :math:`C_n` curve is computed as follows:
+   
+.. math::
+       C_{n}^{Slope}\left(\frac{\alpha_{i+1} + \alpha_i}{2}\right) = \frac{C_n(\alpha_{i+1}) - C_n(\alpha_i)}{\alpha_{i+1} - \alpha_i}
+
+:math:`C_{n,smooth}^{Slope}` is a smoothed version of :math:`C_{n}^{Slope}`, calculated using a triweight kernel with a window of 2 degrees.
+
+
+.. math::
+       C_{l}^{Slope}\left(\frac{\alpha_{i+1} + \alpha_i}{2}\right) = \frac{C_l(\alpha_{i+1}) - C_l(\alpha_i)}{\alpha_{i+1} - \alpha_i}
+
+
+Using :math:`C_{n,smooth}^{Slope}`, ``alphaUpper`` and ``alphaLower`` are computed:
+
+``alphaUpper`` is the smallest angle of attack value between :math:`\alpha_{c_{d0}}` and 20 degrees where the :math:`C_{n,smooth}^{Slope}` curve has started to decrease to 90% of its maximum slope. 
+
+.. math::
+       C_{n,smooth}^{Slope}\left(\alpha^{Upper}\right) < 0.9 \max_{\alpha \in \left[\alpha_{c_{d0}}, \alpha^{Upper}\right]}  C_{n,smooth}^{Slope}\left( \alpha \right) 
+
+
+``alphaLower`` is the largest angle of attack value between -20 degrees and :math:`\alpha_{c_{d0}}` where the :math:`C_{n,smooth}^{Slope}` curve has started to decrease to 90% of its maximum slope. 
+
+.. math::
+       C_{n,smooth}^{Slope}\left(\alpha^{Lower}\right) < 0.9 \max_{\alpha \in \left[\alpha^{Lower}, \alpha_{c_{d0}}\right]}  C_{n,smooth}^{Slope}\left( \alpha \right) 
+
+``Cn1`` is the value of :math:`C_n(\alpha)` at the smallest value of :math:`\alpha` where :math:`\alpha >= \alpha^{Upper}` and the separation function, :math:`f_{st}(\alpha)` = 0.7.
+
+``Cn2`` is the value of :math:`C_n(\alpha)` at the largest value of :math:`\alpha` where :math:`\alpha <= \alpha^{Lower}` and the separation function, :math:`f_{st}(\alpha)`  = 0.7.
+  
+``Cn_offset`` is the average value of the :math:`C_n` curve at ``alphaUpper`` and ``alphaLower``:
+
+.. math::
+       C_{n}^{offset} = \frac{C_n\left(\alpha^{Lower}\right) + C_n\left(\alpha^{Upper}\right)}{2}
+
+``C_nalpha`` is defined as the maximum slope of the smoothed :math:`C_n` curve, :math:`C_{n,smooth}^{Slope}` between :math:`\pm20` degrees angle of attack.
```

Comment:
From the definition of Cnalpha or Clalpha it seems the local slope is used (between i and i+1). I wonder if it would be better to use the slope between i and i0 where i0 if the index of alpha0. That the definition used in the Riso-R-1354 report equation 16.

Reply:
We could change that. I was trying to make it somewhat consistent with the method for computing `C_lalpha`, but it's used in different UA models, so could certainly be done differently.